assets/src/hooks/usePopoverEl.js

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react'
+
+function usePopoverEl () {
```

Comment:
It's probably debatable whether the custom hook is necessary, but the ability to just use `clearPopoverEl` in several places made it seem worth it.

Reply:
Also, I don't love `PopoverEl`, since it's really an object that also includes the key, but I didn't want to disrupt the existing language too much.

Reply:
I like that you extracted it out into a custom hook. Makes it easier to understand what's going on